<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_left">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>71</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/myicon/actions/gtk-go-back-ltr.png</normaloff>:/myicon/actions/gtk-go-back-ltr.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>25</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/myicon/actions/go-up.png</normaloff>:/myicon/actions/go-up.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>117</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/myicon/actions/gtk-go-forward-ltr.png</normaloff>:/myicon/actions/gtk-go-forward-ltr.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_down">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>163</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/myicon/actions/gtk-go-down.png</normaloff>:/myicon/actions/gtk-go-down.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>387</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
